packages/core/src/audio/AudioManager.ts

```
@@ -0,0 +1,50 @@
+      return;
+    }
+    AudioManager._context.resume().then(() => {
+      if (AudioManager._context.state === "running") {
+        window.document.removeEventListener("pointerdown", AudioManager._unlock, true);
+        window.document.removeEventListener("touchend", AudioManager._unlock, true);
+        window.document.removeEventListener("touchstart", AudioManager._unlock, true);
+        AudioManager._unlocked = true;
+      }
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)